packages/experimental/Badge/src/Badge.styling.ts

```diff
@@ -1,18 +1,19 @@
 import { badgeName, BadgeCoreTokens, BadgeTokens, BadgeSlotProps, BadgeProps, BadgeColors, BadgeSizes, BadgeShapes } from './Badge.types';
 import { UseStylingOptions, buildProps, Theme } from '@fluentui-react-native/framework';
-import { borderStyles, layoutStyles } from '@fluentui-react-native/tokens';
+import { borderStyles, layoutStyles, fontStyles } from '@fluentui-react-native/tokens';
 import { defaultBadgeTokens } from './BadgeTokens';
 import { defaultBadgeColorTokens } from './BadgeColorTokens';
 
 export const coreBadgeStates: (keyof BadgeCoreTokens)[] = [
   ...BadgeSizes,
   ...BadgeShapes,
+  'bottom',
   'iconColor',
   'iconSize',
-  'top',
-  'right',
-  'bottom',
   'left',
+  'right',
+  'textPadding',
```

Comment:
Why is this a state? I don't think textPadding controls a whole set of BadgeTokens. Same question with bottom/right/left/top/width

Reply:
Yeah, you're  right. I've added this automatically before. Removed not needed tokens